spec/tasks/generate_fake_users_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+describe 'rake generate_fake_users', type: :task do
+  it 'preloads the Rails environment' do
+    expect(task.prerequisites).to include 'environment'
+  end
+  it 'runs gracefully with no subscribers' do
+    expect { task.execute }.not_to raise_error
+  end
+
+  context 'Less than 10 users' do
+    it 'should generate 10 fake users' do
+      expect { task.execute }.to change { User.count }.from(0).to(10)
```

Comment:
Don't we always create an admin in `seeds.rb` and therefore can't be sure we have 0 users?

Reply:
Yeah, but the database is cleaned for each tests. Seeds.rb should not effect a test.

Reply:
Ok, I wasn't sure.